services/rule_engine/src/main/java/agents/Moli.java

```diff
@@ -21,15 +27,68 @@ public class Moli extends Agent {
     public void showHelloScreen() {
         ArmoryAPI.show(
             getConnection("armory"),
-            new Screen ("Hello")
-                .add(new Title("Hello world!"))
+            new Screen("Welcome")
+                .add(new Header("logo.png", false))
+                .add(new Title("Bok, ja sam Moli! Može anketa?"))
+                .add(new Description("Hvala ti puno što mi pomažeš <3"))
+                .add(new SubmitButton("welcomeStarted", "Idemo!", "askForGender")),
+            new Screen("askForGender")
+                .add(new Header("logo.png", true)) 
+                .add(new Title("Jesi li miočanin ili miočanka?")) 
+                .add(new SubmitButton("askForGenderStarted", "Miočanin", "askForAgem")) //uhvati podatke i salji Ravnatelj agentu
+                .add(new SubmitButton("askForGenderStarted", "Miočanka", "askForAgef")),
+            new Screen("askForAgem") //postoje m i f verzije stranica sa prilagodenim recenicama s obzirom na spol
+                .add(new Header("logo.png", true)) 
+                .add(new Title("Koji si razred?"))
+                .add(new SubmitButton("askForAgemStarted", "Prvi", "askForRatingm")) //uhvati podatke i salji Ravnatelj agentu
```

Comment:
Do you think we can just use `askForAgeStarted` instead `askForAgemStarted`? That way the code will be bit simpler

Reply:
I agree, currently there are no gender-specific words used on those pages, so it can be simplified